pex/bin/pex.py

```diff
@@ -525,6 +531,11 @@ def build_pex(args, options, resolver_option_builder):
       for interpreter in options.python or [None]
     ]
 
+  if options.interpreter_constraints:
+    safe_constraints = options.interpreter_constraints.strip("'").strip('"')
```

Comment:
I added this to avoid issues with using < chars on the command line; this issue came up during integration testing.